src/components/layouts/AppLayout/AppLayout.jsx

```diff
@@ -10,15 +10,11 @@ const AppLayout = () => {
   return (
     <>
       <Header />
-      <Suspense
-        fallback={
-          <Main>
-            <AppLoader />
-          </Main>
-        }
-      >
-        <Outlet />
-      </Suspense>
+      <Main>
```

Comment:
Leave it as it was before.

Reply:
No)) Do you see how jump your footer when we submit forms etc.? We need Main here, not separate in each page .

Reply:
I closed now, but on meet i will show what i mean.